pkg/ext/openapi.go

```diff
@@ -0,0 +1,2661 @@
+package ext
+
+import (
+	generatedopenapi "github.com/rancher/rancher/pkg/generated/openapi"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	common "k8s.io/kube-openapi/pkg/common"
+	spec "k8s.io/kube-openapi/pkg/validation/spec"
+)
+
+// This file can be removed once we have at least one type that is generating OpenAPI spec.
+func getOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
```

Comment:
The k8s.io/apiserver library expects at least these to be defined even though we're not implementing a store yet. Not ideal but once Collie implements their token then this file can be removed.